lib/block-supports/layout.php

```diff
@@ -237,19 +262,23 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 	$has_block_gap_support  = isset( $block_gap ) ? null !== $block_gap : false;
 	$default_block_layout   = _wp_array_get( $block_type->supports, array( '__experimentalLayout', 'default' ), array() );
 	$used_layout            = isset( $block['attrs']['layout'] ) ? $block['attrs']['layout'] : $default_block_layout;
+	$class_names            = array();
+	$layout_definitions     = _wp_array_get( $global_layout_settings, array( 'definitions' ), array() );
+	$block_classname        = wp_get_block_default_classname( $block['blockName'] );
+	$container_class        = wp_unique_id( 'wp-container-' );
+	$layout_classname       = '';
+	$use_global_padding     = gutenberg_get_global_settings( array( 'useRootPaddingAwareAlignments' ) ) && ( isset( $used_layout['inherit'] ) && $used_layout['inherit'] || isset( $used_layout['contentSize'] ) && $used_layout['contentSize'] );
+
 	if ( isset( $used_layout['inherit'] ) && $used_layout['inherit'] ) {
```

Comment:
a nit, could this be moved up to just after the $used_layout declaration as it seems like the other declarations are not needed if this returns early?
```suggestion
	$global_layout_settings = gutenberg_get_global_settings( array( 'layout' ) );
	$default_block_layout    = _wp_array_get( $block_type->supports, array( '__experimentalLayout', 'default' ), array() );
	$used_layout                   = isset( $block['attrs']['layout'] ) ? $block['attrs']['layout'] : $default_block_layout;

	if ( isset( $used_layout['inherit'] ) && $used_layout['inherit'] ) {
		if ( ! $global_layout_settings ) {
			return $block_content;
		}
	}

	$block_gap                          = gutenberg_get_global_settings( array( 'spacing', 'blockGap' ) );
	$has_block_gap_support  = isset( $block_gap ) ? null !== $block_gap : false;
	$class_names                     = array();
	$layout_definitions             = _wp_array_get( $global_layout_settings, array( 'definitions' ), array() );
	$block_classname              = wp_get_block_default_classname( $block['blockName'] );
	$container_class                 = wp_unique_id( 'wp-container-' );
	$layout_classname             = '';
	$use_global_padding         = gutenberg_get_global_settings( array( 'useRootPaddingAwareAlignments' ) ) && ( isset( $used_layout['inherit'] ) && $used_layout['inherit'] || isset( $used_layout['contentSize'] ) && $used_layout['contentSize'] );
```

Reply:
Yeah that makes sense!